framework/test/grt-eval-tab4-unit.sh

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env bash
```

Comment:
Please document the difference between `grt-eval-tab4.sh` and `grt-eval-tab4-unit.sh`.  The comments at the top of their files are identical, and much of their code is identical too.